solidus_affirm.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*", 'LICENSE', 'Rakefile', 'README.md']
   s.test_files = Dir['test/**/*']
 
-  s.add_dependency 'solidus', ['>= 1.1', '< 3']
+  s.add_dependency 'solidus', ['>= 1.4', '< 3']
```

Comment:
Why was this decision made? The commit message doesn't indicate why this had to be done.